lib/libc/newlib/libc-hooks.c

```diff
@@ -23,6 +23,8 @@
 #include <zephyr/sys/mem_manage.h>
 #include <sys/time.h>
 
+#include <newlib.h>
```

Comment:
Surely one of the above headers already includes this? Odd, if not. But, nbd in any case.